test/Integration/Builder/Blueprint/Factory/CodeGenerator/Store/FilesystemTest.php

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types = 1);
+
+namespace RstGroup\ObjectBuilder\Test\Integration\Builder\Blueprint\Factory\CodeGenerator\Store;
+
+use PHPUnit\Framework\TestCase;
+use RstGroup\ObjectBuilder\Builder\Blueprint\Factory\CodeGenerator\PatternStore\Filesystem;
+
+class FilesystemTest extends TestCase
```

Comment:
There are libs to mock file system

Reply:
Yes, but is it good cause to use his? In integration test I don't want mock object of my integration - filesystem.